bundles/org.openhab.binding.nuki/src/main/java/org/openhab/binding/nuki/internal/NukiHandlerFactory.java

```diff
@@ -92,19 +92,16 @@ public boolean supportsThingType(ThingTypeUID thingTypeUID) {
     public void unregisterHandler(Thing thing) {
         super.unregisterHandler(thing);
         logger.trace("NukiHandlerFactory:unregisterHandler({})", thing);
-        if (thing.getHandler() instanceof NukiBridgeHandler && nukiApiServlet != null) {
-            nukiApiServlet.remove((NukiBridgeHandler) thing.getHandler());
-            if (nukiApiServlet.countNukiBridgeHandlers() == 0) {
-                nukiApiServlet = null;
-            }
+        @Nullable
+        ThingHandler handler = thing.getHandler();
```

Comment:
The compiler is more intelligent on local variables. Therefore, the Nullable could be removed.